src/static/hyperhtml/examples/example/react/not-primitive-props/hyper.js

```diff
@@ -0,0 +1,26 @@
+customElements.define(
+  'h-welcome',
+  class HyperWelcome extends HTMLElement {
+    constructor(...args) {
+      super(...args);
+      this.html = hyperHTML.bind(this);
+    }
```

Comment:
I'd rather go:

```js
get html() { return this._html || (this._html = hyperHTML(this)); }
```

and drop the constructor dance. I also would drop the `connectedCallback` since it's not needed if you render when the component receive the user. It wouldn't know what to render otherwise.